src/js/components/DateInput/DateInput.js

```diff
@@ -74,8 +74,12 @@ const DateInput = forwardRef(
     useEffect(() => {
       if (
         schema &&
-        value &&
-        ((Array.isArray(value) && value[0]) || !Array.isArray(value))
+        (value || value?.length === 0) &&
+        (
+          (Array.isArray(value) && value[0])
+          || !Array.isArray(value)
+          || value?.length === 0
```

Comment:
I think was checked on line 77 already. I suspect you mean `value[0]?.length === 0`? In which case, I think line 79 could be: `(Array.isArray(value && value[0] !== undefined) || !Array.isArray(value)`

Reply:
Hello, thank you for the review.
Unfortunately, this will not work because I want to reset the date when I pass an `[ ]` as a value. And in this case `value[0] !== undefined` will be false.